### DISCUSSION TITLE
module 'jdtls_setup' not found:

### DISCUSSION THREAD

**Discussion by thegxi** [mfussenegger/nvim-jdtls#297] [General]
<img width="275" alt="image" src="https://user-images.githubusercontent.com/43883006/179945508-dd26cd4d-b6ab-4883-8bf2-76b53f8c7215.png">
<img width="671" alt="image" src="https://user-images.githubusercontent.com/43883006/179945963-cb373c10-c1a0-4439-ad2b-f5efbd61308e.png">
<img width="729" alt="image" src="https://user-images.githubusercontent.com/43883006/179946114-f9393f00-8341-4333-b79f-e907ba5a0705.png">
有遇到同样的情况的吗？我照着REMDME设置的，lua找不到jdtls_setup模块

**Comment by mfussenegger** [comment]
Seems like you have a `require('jdtls_setup')` call somewhere, the correct module name is `jdtls.setup`.
